modules/history/src/views/full/ConversationPicker.jsx

```diff
@@ -54,7 +51,9 @@ export class ConversationPicker extends React.Component {
     const actions = [
       {
         icon: <Icon icon="refresh" />,
-        onClick: this.props.refresh
+        onClick: this.props.refresh,
+        tooltip: 'refresh conversations',
+        tooltipPosition: Position.BOTTOM
```

Comment:
Let's keep the standard position, unless it cause some issue with rendering. Either we use all right or bottom positions, but it shouldn't be chosen by individual modules. Also, you could use casing for text, like Refresh Conversations, which is a bit better to read than refresh conversations

Reply:
Yes it does cause some issues with rendering. When on the right, it completly hides the toolbar. 
![image](https://user-images.githubusercontent.com/25970722/59607835-11f83a80-90e2-11e9-8088-54bd00fbf362.png)

Reply:
I agree, but it only hides it when the mouse is over it. I was thinking that it's preferable to hide the toolbar content instead of the content of the side panel where the button is located. I find the tooltip more natural from the left or right side, but if we want to change it, i'm open to that, but it should be identical everywhere. The content page & flow builder behaves the same and also have a toolbar